Eshop/apps/orders/models.py

```diff
@@ -52,10 +54,10 @@ class OrderItem(models.Model):
 
     @property
     def sell_price(self):
```

Comment:
fix date_of_birth if none
add @receiver(post_save, sender=OrderItem)
add tests for discounts